src/ts/components/core/combobox/MultiSelect.tsx

```diff
@@ -96,10 +101,15 @@ const MultiSelect = (props: Props) => {
     const handleBlur = () => {
         setProps({
             n_blur: n_blur + 1,
-            ...(debounce === true && { value: selected })
+            ...(debounce === true && { value: selected }),
         });
     };
 
+    const handleSearchChange = (newSearchVal) => {
+        setSearchVal(newSearchVal);
+        setProps({ searchValue: newSearchVal });
```

Comment:
If you're setting both of these simultaneously now, do we even need to have `searchValue` copied into state? Can't we just use the prop directly?

Reply:
I deleted the `setSearchVal(newSearchVal);` and it works :-)

Reply:
I was making a stronger statement actually - that maybe now that `handleSearchChange` is updating the prop we don't need
```py
const [searchVal, setSearchVal] = useState(searchValue);
```
or the rest of the state management it entails at all, and we could just forward `searchValue` directly to the component as `searchValue={searchValue}` or even leave it in `others`.

Reply:
oh, cool!  That simplifies things a lot!